### PR TITLE
1st working version of rebuilt SearchNav

### PR DESCRIPTION
**Pull request by Mxx1029**
So, this is working, but without the date picker from Maria. 
So I suggest you have a look at it, but merge Marias PR first.
Maybe wait with merging this one until I'm back. Also if you have comments. 
I left everything old commented out, and always created copies of every line I changed, so it will be confusing to make sense of. 
Happy to explain everything in detail :)

### CODE REVIEW

File: src/components/LandingPage/LandingPage.jsx [hunk 1]
```diff
@@ -1,64 +1,99 @@
...
+				<SearchNav
+					events={events}
+					setEvents={setEvents}
+                    category={category}
+					setCategory={setCategory}
+                    date={date}
+					setDate={setDate}
+                    keyword={keyword}
+					setKeyword={setKeyword}
+                    getSearchResults={getSearchResults}
```
Comment (mariagluck) [inline src/components/LandingPage/LandingPage.jsx:79]: is this like a funky prettier or beautify code formating? they are not very aligned,...just wondering :)))

File: src/components/SearchNav/KeywordSearch.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+			{/* <div>
+                    <ul>
+                        {filteredEvents.map((event) => (
+                            <li key={event.id}>{event.name}</li>
+                        ))}
+                    </ul>
+            </div> */}
+		</>
+	);
+}
```
Comment (mariagluck) [inline src/components/SearchNav/KeywordSearch.jsx:51]: this is the new girl in town! I see...:)

File: src/components/SearchNav/SearchNav.jsx [hunk 3]
```diff
@@ -1,53 +1,77 @@
...
+import DropdownDate from "./DropdownDate.jsx";
+import KeywordSearch from "./KeywordSearch.jsx";
+
+// export default function SearchNav(props) {
+export default function SearchNav({
+    events,
+	category,
+	setCategory,
+	date,
+	setDate,
```
Comment (mariagluck) [inline src/components/SearchNav/SearchNav.jsx:14]: again the prettier format? should we align it or not...? who knows!!

**Review by mariagluck — APPROVED** [review]

File: src/components/LandingPage/LandingPage.jsx (hunk 1 above)
Comment (Mxx1029) [reply src/components/LandingPage/LandingPage.jsx:79]: apps, didn't use prettier maybe. Will do before merging...

File: src/components/SearchNav/KeywordSearch.jsx (hunk 2 above)
Comment (Mxx1029) [reply src/components/SearchNav/KeywordSearch.jsx:51]: exactly :)

File: src/components/SearchNav/SearchNav.jsx (hunk 3 above)
Comment (Mxx1029) [reply src/components/SearchNav/SearchNav.jsx:14]: i'm gonna use prettier again. let's hope that helps :) It looks ok on my machine...

**Pull request merged**